### DISCUSSION TITLE
RG35XX H

### DISCUSSION THREAD

**Discussion by benchristey** [rg35xx-cfw/rg35xx-cfw.github.io#68] [Q&A]
Hey! Very excited for the Plus version to start getting some love - I was wondering if the plus firmware would "just work" for the H model or if there will be some nuances?

**Comment by acmeplus** [comment]
It should mostly work, but I don't have an H yet. There are some very small differences (control related) but for the most part they should be compatible. You should be able to flash the rg35xx plus version to an sdcard and try it.

**Reply by benchristey** [reply]
Excellent! Mine is en route - currently at the "Departed from country of origin" stage. I'll give it a go!

Am I correct in thinking this CFW supports Pygame?
